container_api/src/container.rs

```diff
@@ -0,0 +1,233 @@
+use config::Configuration;
+use holochain_cas_implementations::{
+    cas::file::FilesystemStorage, eav::file::EavFileStorage, path::create_path_if_not_exists,
+};
+use holochain_core::context::Context;
+use holochain_core_types::{error::HolochainError, json::JsonString};
+use holochain_dna::Dna;
+use Holochain;
+
+use holochain_core::{logger::Logger, persister::SimplePersister};
+use holochain_core_types::entry::agent::Agent;
+use std::{
+    collections::HashMap,
+    convert::TryFrom,
+    fs::File,
+    io::prelude::*,
+    sync::{Arc, Mutex, RwLock},
+};
+
+use boolinator::*;
+
+pub struct Container {
+    pub instances: HashMap<String, Holochain>,
+}
+
+impl Container {
+    pub fn new() -> Self {
+        Container {
+            instances: HashMap::new(),
+        }
+    }
+
+    pub fn start_all(&mut self) {
+        let _ = self.instances.iter_mut().for_each(|(id, hc)| {
+            println!("Starting instance \"{}\"...", id);
```

Comment:
I think we shouldn't really be assuming that the container wants to print stuff.  If anything it should be a log or a console or something like that.  This is fine for now, but we should log a followup (or put it in the tree)

Reply:
Yeah, container_api should maybe not and I will remove this in future PRs. Felt right to have the container executable print to std-out which can be redirected by the shell context that runs the process.